src/streamlink/plugins/canalplus.py

```diff
@@ -55,9 +55,9 @@ def _get_streams(self):
         parsed = []
         headers = {'User-Agent': self._user_agent}
 
-        # Some videos may be also available on Dailymotion (especially on CNews)
+        # Some videos may be also available on  (especially on CNews)
         if videos['ID_DM'] != '':
-            for stream in self.session.streams('https://www.dailymotion.com/video/' + videos['ID_DM']).items():
+            for stream in self.session.streams('https://www..com/video/' + videos['ID_DM']).items():
```

Comment:
This looks like you've accidentially replaced the string "DailyMotion" with "" in the entire project.

Reply:
Yes, a single file was accidentally modified. Sorry. Thanks for the fix.

Reply:
Which fix? I've requested a change.

Reply:
I undoed the change made by mistake.